csharp/src/ProtocolBuffers/FieldAccess/SinglePrimitiveAccessor.cs

```diff
@@ -42,6 +43,7 @@ internal class SinglePrimitiveAccessor<TMessage, TBuilder> : IFieldAccessor<TMes
         where TBuilder : IBuilder<TMessage, TBuilder>
     {
         private readonly Type clrType;
+        private readonly FieldDescriptor field;

```

Comment:
Nit: we don't need this field any more (as we're just capturing the descriptor in the delegate where we need it).